Makefile

```diff
@@ -170,6 +194,51 @@ define proto_compile
 
 endef
 
+.PHONY: lint
+lint: lint-imports lint-nocommit lint-license lint-go
+
+.PHONY: lint-go
+lint-go: $(LINT)
+	$(LINT) -v run
+
+.PHONY: lint-license
+lint-license:
+	@echo Verifying that all files have license headers
+	@mkdir -p .scripts/lint
+	@curl -s -o .scripts/lint/updateLicense.py https://raw.githubusercontent.com/jaegertracing/jaeger/main/scripts/lint/updateLicense.py
+	@chmod +x .scripts/lint/updateLicense.py
+	@./.scripts/lint/updateLicense.py $(ALL_SRC) $(SCRIPTS_SRC) > $(FMT_LOG)
+	@[ -s "$(FMT_LOG)" ] || echo "✅ All files have license headers"
+
+
+.PHONY: lint-nocommit 
+lint-nocommit:
+	@if git diff origin/main | grep '@no''commit' ; then \
+		echo "❌ Cannot merge PR that contains @no""commit string" ; \
+		false ; \
+	fi
+
+.PHONY: lint-imports
+lint-imports:
+	@echo Verifying that all files have correctly ordered imports
+	@mkdir -p .scripts/lint
+	@curl -s -o .scripts/lint/import-order-cleanup.py https://raw.githubusercontent.com/jaegertracing/jaeger/main/scripts/lint/import-order-cleanup.py
```

Comment:
this should be done in a separate target that is a dependency of targets that invoke scripts. `make fmt` below would not work on its own.